repose-aggregator/core/repose-core-api/src/test/scala/org/openrepose/core/systemmodel/SystemModelSchemaTest.scala

```diff
@@ -0,0 +1,278 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2015 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+
+package org.openrepose.core.systemmodel
+
+import org.junit.runner.RunWith
+import org.openrepose.commons.test.ConfigValidator
+import org.scalatest.{BeforeAndAfter, FunSpec, Matchers}
+import org.scalatest.junit.JUnitRunner
+import org.xml.sax.SAXParseException
+
+@RunWith(classOf[JUnitRunner])
+class SystemModelSchemaTest extends FunSpec with BeforeAndAfter with Matchers {
+  var validator: ConfigValidator = _
+
+  before {
+    // for some reason this class requires it to be reinitialized for each test, otherwise it complains about ID reuse
```

Comment:
Quite.  I checked it like 5 times.  Once a repose-cluster id had been used in one config, it wasn't allowed to be used again in another config.